go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/google/go-cmp v0.6.0
-	github.com/hashicorp/terraform-plugin-codegen-spec v0.1.0
+	github.com/hashicorp/terraform-plugin-codegen-spec v0.1.1-0.20231017164028-264c2bd37f5f
```

Comment:
This will be updated to `v0.2.0` once https://github.com/hashicorp/terraform-plugin-codegen-spec/pull/55 is merged and the `v0.2.0` release is cut.

Reply:
We chatted about this earlier, but maybe we can wait until `codegen-framework` needs a `v0.2.0` release and then re-assess how we want to bump `codegen-spec`

Reply:
Ok, we'll stick with using SHAs until we want to release 👍